pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -2877,6 +2877,23 @@ type CephFilesystemSubVolumeGroupSpec struct {
 	// list of Ceph Filesystem volumes with `ceph fs volume ls`. To learn more about Ceph Filesystem
 	// abstractions see https://docs.ceph.com/en/latest/cephfs/fs-volumes/#fs-volumes-and-subvolumes
 	FilesystemName string `json:"filesystemName"`
+	// Pinning configuration of CephFilesystemSubVolumeGroup,
+	// reference https://docs.ceph.com/en/quincy/cephfs/fs-volumes/#pinning-subvolumes-and-subvolume-groups
+	// +optional
+	Pinning CephFilesystemSubVolumeGroupSpecPinning `json:"pinning,omitempty"`
+}
+
+// CephFilesystemSubVolumeGroupSpecPinning represents the pinning configuration of SubVolumeGroup
+type CephFilesystemSubVolumeGroupSpecPinning struct {
+	// Pinning type of CephFilesystemSubVolumeGroup
+	// +optional
+	// +kubebuilder:default=distributed
+	// +kubebuilder:validation:Enum=random;export;distributed
+	PinType string `json:"pinType,omitempty"`
```

Comment:
We are already inside the context of the `pinning` element, so I'd suggest no need for the "pin" prefix on the settings.
```suggestion
	Type string `json:"type,omitempty"`
```